lib/plugins/aws/package/compile/events/s3/index.js

```diff
@@ -21,7 +45,10 @@ class AwsCompileS3Events {
 
   newS3Buckets() {
     const bucketsLambdaConfigurations = {};
+    const bucketsLogicalIds = {};
```

Comment:
Name suggests it's a logicalIds map, while it's something more, let's name it then maybe `bucketsMeta`